test/sanitize-html.test.js

```diff
@@ -145,7 +145,11 @@ describe('Sanitize HTML', () => {
   });
 
   test('twitch.tv embedded content should not be removed', () => {
-    const data = sanitizeHTML('<iframe src="https://www.twitch.tv/0pensrc"></iframe>');
-    expect(data).toBe('<iframe src="https://www.twitch.tv/0pensrc"></iframe>');
+    const data = sanitizeHTML(
+      '<iframe src="https://player.twitch.tv/?video=1303803789&parent=dev.to"></iframe>'
```

Comment:
According to [this](https://dev.twitch.tv/docs/embed/video-and-clips/), `parent` is the domain that will embed the stream. So this should be telescope.cdot.systems vs dev.to. I'd say we have to take a twitch iframe, and manually add a `parent` attribute to it.
Probably will solve the problem @manekenpix  mentioned [here](https://github.com/Seneca-CDOT/telescope/pull/3023#issuecomment-1047363874)

Reply:
Will this also affect the host domain if ran in a different environment like local or staging? Was thinking of just grabbing that from the `.env` and using that here. Thoughts?

Reply:
That's what I think too, we can grab the value it from .env